feature-configs/base/sriov/06-ipv6-network_attachment_defintion.yaml

```diff
@@ -7,3 +7,13 @@ metadata:
   namespace: sriov-testing
 spec:
   config: '{ "cniVersion":"0.3.1", "name":"test", "type":"sriov", "vlan":0,"vlanQoS":0,"ipam": {"type": "host-local","ranges": [[{"subnet": "3ffe:ffff:0:01ff::/64"}], [{"subnet": "10.10.0.0/24"}]  ],"dataDir": "/run/my-orchestrator/container-ipam-state3"}}'
+---
```

Comment:
Hi @mmirecki I probably miss this in your previews PR.

Can you please move this file under the `demo` folder and not the `base`.
You will also need to update the `kustomization.yaml` in both directories.

Reply:
Done